index.js

```diff
@@ -23,9 +25,11 @@ var ERROR_TYPES = {
 };
 
 /**
- * Constructor you must pass in a config object with the following properties set:
+ * Constructor you must pass in an appId string identifying your app, plus a config object with the
+ * following properties set:
  *
  * mandatory params
+ * appUA = "my_app"
```

Comment:
Is app version not also expected here? Should this not be `appUA = "my_app/version"`